src/_guides/language/analysis-options.md

```diff
@@ -204,6 +204,50 @@ For example, some rules are more appropriate for library packages
 and others are designed for Flutter apps.
 Note that linter rules can have false positives, unlike static analysis.
 
+### Enabling Dart team recommended rules: lints {#lints}
+
+The Dart team provides two sets of recommended linter rules.
+
+Two sets of lints are provided by the package:
```

Comment:
This is a bit repeated with the previous sentence.

Reply:
Good point, fixed with suggestions by @kwalrath :)